File: LLM-Evaluation/src/main/org/cornell/Task.java
package org.cornell;

import java.time.LocalDate;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Task {

    private int id;
    private String owner;
    private String title;
    private String description;
    private LocalDate startDate;
    private LocalDate endDate;
    private LocalDate completionDate;
    private boolean completed;

    public Task(int id, String owner, String name, String description, LocalDate startDate, LocalDate endDate) {
        this.id = id;
        this.owner = owner;
        this.title = name;
        this.description = description;
        this.startDate = startDate;
        this.endDate = endDate;
        this.completionDate = null;
        this.completed = false;
    }

    public void markAsCompleted() {
        this.completed = true;
    }
}

File: translation/2024-11-20_13-10-45/run1/Task.py
import datetime

class Task:

    def __init__(self, id, owner, name, description, startDate, endDate):
        self.id = id
        self.owner = owner
        self.title = name
        self.description = description
        self.startDate = startDate
        self.endDate = endDate
        self.completionDate = None
        self.completed = False

    def markAsCompleted(self):
        self.completed = True
